ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/event_handlers.schema.yml

```diff
@@ -42,6 +42,55 @@ keys:
           - on-boot
           - on-logging
           - on-startup-config
+          - on-maintenance
+        trigger_on_maintenance:
+          description: Settings required for trigger 'on-maintenance'.
+          type: dict
+          keys:
+            operation:
+              type: str
+              required: true
+              valid_values:
+              - "enter"
+              - "exit"
+            bgp:
+              description: Trigger condition occurs on maintenance operation of specified BGP peer.
+              type: dict
+              keys:
+                neighbor_address_ipv4:
+                  type: str
+                neighbor_address_ipv6:
+                  type: str
+                peer_group_name:
+                  type: str
+            stage:
+              type: str
+              required: true
+              description: Stages of maintenance operation. "vrf" stage can be defined for "bgp" only.
+              valid_values:
+              - after
+              - before
+              - all
+              - begin
+              - end
+              - vrf
+            after_before_stage:
+              type: str
+              description: Action is triggered after/before specified stage.
+              valid_values:
+              - "bgp"
+              - "linkdown"
+              - "mlag"
+              - "ratemon"
```

Comment:
bgp, linkdown, mlag and ratemon are stages and after, before etc are action so i think we should update the key accordingly.

Reply:
any suggestion fir the key_name?

Reply:
fixed